README.md

```diff
@@ -1,18 +1,56 @@
 
-This project aims to create an Open source scheduling stack for distributed energy resources (DER) control according to IEC 61850-90-10.
+## Targeted applicability
+- DER = Distributed Energy Resources (e.g. PV solar, Windpower, Battery infeed into distribution grid)
+- EMS = (Plant / Industrial) Energy Managment Systems
+- HEMS = Home Energy Management Systems
+- Specific solutions such as the [FNN-steurbox](https://www.vde.com/de/fnn/arbeitsgebiete/digitalisierung-metering/lastenhefte/steuerbox) in Germany
 
-The main functionalities will be to control a DERs regarding:
+## Why IEC scheduling?
+IEC scheduling solves two structural problems in controling complex energy systems where more than one actor and more than one use-case is involved (*...and that is nearly almost the case*).
+
+**Problem 1**: There are many controls from remote, and in the local device itself. How do I get myself into control what is happening with all those loose controls? Otherwise, I won't understand the behavior and lose control.
+
+![Several actors with several capacity envelopes](./images/readme_totalenvelopes.png)
+
+
+**Answer**: IEC scheduling completely changes the concept of controlling, compared to what is known till sofar in the IEC landscape for energy automation. The main principle of it is to place ALL controls into parallell schedules and where a policy defines the priority of those schedules. There are NO direct controls. All controls, even locally generated are placed in (separate) schedules. Included in defining this policy is to give priority to schedules dependent on situational circumstances, such as:
+1. Device just starting up, so soft-start may get priorities
+2. (Internet)communication is down, so slowly move towards pre-defined (safe) schedules.
+3. Voltage is too high, so curtailment of infeed get priority (and parallel simulate with HEMS to use more energy locally)
+
+![Several actors with several IEC-schedules](./images/readme_totalschedules.png)
+
+**Problem 2**:  What communication protocol to use, because my use-case needs "protocol y", my DSO needs "protocol z" and for car-charging I need another one. Please don't create or force me to use another communication protocol, do you?
+**Answer**: Although the standard for Scheduling is under the IEC 61850 standard, however the funcion scheduling itself isn't requiering the IEC 61850 communication protocol use.  It doesn't matter as long as you can manage a schedule to the scheduling logic, whatever protocol you use. This will enable you to choose your protocol which is needed for your use-case. And yes, other requirements to realize the right security level should be applied to all protocols.
```

Comment:
```suggestion
**Answer**: Although the standard for Scheduling is under the IEC 61850 standard, the funcion scheduling itself isn't requiering the IEC 61850 communication protocol use.  It doesn't matter as long as you can manage a schedule to the scheduling logic, whatever protocol you use. This will enable you to choose your protocol which is needed for your use-case. And yes, other requirements to realize the right security level should be applied to all protocols.
```